test/modules/model/TinyLlamaWithFusedAttention/model.py

```diff
@@ -0,0 +1,87 @@
+import torch
+
+from tico.serialize.operators.adapters.llama_rmsnorm import patched_llama_rmsnorm
+from tico.serialize.operators.adapters.onert.llama_attention import (
+    llama_attention_forward_adapter,
+)
+from tico.utils.pytree_utils import register_dynamic_cache
+from tico.utils.record_input import RecordingInput
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from transformers.models.llama.modeling_llama import LlamaAttention
+
+from test.modules.base import TestModuleBase
+from test.utils import tag
+
+
+@tag.use_onert
```

Comment:
@dayo09 With this tag, it tries to run using `onert`. However, currently it gets error since `onert` does not allow _batchmatmul lhs const_ 😢 .

Could you please kindly teach me a way to run your pass https://github.com/Samsung/TICO/pull/359 before test framework running `onert`?